ts/Data/DataTableCore.ts

```diff
@@ -27,10 +27,10 @@ import type DataEvent from './DataEvent.js';
 import type DataTable from './DataTable.js';
 import type DataTableOptions from './DataTableOptions.js';
 
+import ColumnUtils from './ColumnUtils.js';
```

Comment:
I think we should use local constants for these properties. We don't use the `setLength` function in this module, so when we rewrite to named imports we should be able to tree-shake off some bytes.
```suggestion
import ColumnUtils from './ColumnUtils.js';
const { shift, splice } = ColumnUtils;
```